scripts/lepidopter-sign.sh

```diff
@@ -1,17 +1,16 @@
-#!/bin/sh
-set -e
+#!/bin/bash -e
```

Comment:
I suggest explicit `-e` because otherwise afaik `-e` is not activated if you do `bash $script_name`